lib/beamchmark/utils.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Beamchmark.Utils do
+  # TODO what is the return type?
```

Comment:
I believe it is a `binary/String` [here](https://github.com/elixir-lang/elixir/blob/a64d42f5d3cb6c32752af9d3312897e8cd5bb7ec/lib/elixir/lib/io/ansi.ex#L4)